androidshared/src/main/res/values/toolbars.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <style name="Widget.AndroidShared.Toolbar.Primary.CloseIcon" parent="Widget.MaterialComponents.Toolbar.Primary">
+    <style name="Widget.AndroidShared.Toolbar.Primary.CloseIcon" parent="Widget.MaterialComponents.Toolbar.Surface">
```

Comment:
We still want to keep a "grey" status bar rather than a primary one (which would be bright blue)